repo/packages/P/portworx/0/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name": "portworx",
+  "version": "1.1",
```

Comment:
`version` should be serviceVersion-packagingVersion 
Ex: if portworx version is 1.2.3 then `version` here will be `1.2.3-0.0.1`